### ISSUE TITLE
Categorical Cross Entropy Loss

### ISSUE THREAD

**Issue by farkas93** [farkas93/PatinAI#10]
Decide if you want to implement the same way as in tensorflow:
L(y_true, y_pred) =- sum( y_true * ln(y_pred/ sum(y_pred, axis=0) ) )
and then derivative 
dL_dYpred = -(y_true/y_pred - 1/np.sum(y_pred, axis=0))

**Comment by farkas93** [comment]
Already implemented, tested and resolved with commit  7a17b5f45e5174d3364e7dbdb3ab4a4fa809bf9b
